scripts/linters/pylint_extensions.py

```diff
@@ -171,7 +172,7 @@ class HangingIndentChecker(checkers.BaseChecker):
     of hanging indentation.
     """
 
-    __implements__ = interfaces.IRawChecker
+    __implements__ = interfaces.ITokenChecker
```

Comment:
It is great to see this being changed to TokenChecker.

However, I'm curious -- it looks the check was already there but it wasn't catching the line breaks needed. Do you know why?

Reply:
Because before it is checking only those parentheses which has a comma in them
Example: func(1, 2,3)
and ignoring this type of functions
func(abc(def))